fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/literal/StructLiteral.java

```diff
@@ -89,18 +120,29 @@ public String toString() {
 
     @Override
     public String toSql() {
-        return "{" + fields.stream().map(Literal::toSql).collect(Collectors.joining(",")) + "}";
+        StringBuilder sb = new StringBuilder();
```

Comment:
this func is same with legacyliteral toSqlImpl ? maybe legacyliteral impl is wrong

Reply:
will rewrite them later